connect_ext_ppr/constants.py

```diff
@@ -0,0 +1,362 @@
+# Common schemas
+# Categories
+CATEGORIES_SCHEMA = {
+    "type": "array",
+    "allOf": [
+        {"contains": {"const": "Name_EN"}},
+        {"contains": {"const": "Description_EN"}},
+    ],
+    "items": {
+        "anyOf": [
+            {
+                "pattern": "^(Description|Name)_[a-zA-Z]+$",
+            },
+            {
+                "enum": [
+                    "Name_EN",
+                    "Description_EN",
+                    "ExpandCategory",
+                    "DisplayInCCP",
+                    "ParentCategory",
+                ],
+            },
+        ],
+    },
+    "uniqueItems": True,
+}
+# Policies
+POLICIES_SCHEMA = {
+    "type": "array",
+    "items": {
+        "type": "string",
+        "enum": [
+            "ServicePlan",
+            "Period",
+            "PeriodType",
+            "Trial",
+            "Action",
+            "ActionPeriod",
+            "ActionPeriodTimezone",
+            "ApplicableTo",
+        ],
+    },
+    "allOf": [
+        {"contains": {"const": "ServicePlan"}},
+        {"contains": {"const": "Period"}},
+        {"contains": {"const": "PeriodType"}},
+        {"contains": {"const": "Trial"}},
+        {"contains": {"const": "Action"}},
+        {"contains": {"const": "ActionPeriod"}},
+        {"contains": {"const": "ActionPeriodTimezone"}},
+        {"contains": {"const": "ApplicableTo"}},
+    ],
+    "additionalItems": False,
+}
+
+# Per Sheet Schema
+RESOURCES_SCHEMA = {
+    "Resources": {
+        "type": "array",
+        "items": {
+            "type": "string",
+            "enum": [
+                "Name_EN",
+                "Description_EN",
+                "ResourceCategory",
+                "MPN",
+                "UOM",
+                "Measurable",
+            ],
+        },
+        "allOf": [
+            {"contains": {"const": "Name_EN"}},
+            {"contains": {"const": "Description_EN"}},
+            {"contains": {"const": "ResourceCategory"}},
+            {"contains": {"const": "MPN"}},
+            {"contains": {"const": "UOM"}},
+            {"contains": {"const": "Measurable"}},
+        ],
+        "additionalItems": False,
+    },
+}
+SERVICE_PLANS_SCHEMA = {
+    "ServicePlans": {
+        "type": "array",
+        "items": {
+            "type": "string",
+            "anyOf": [
+                {
+                    "enum": [
+                        "OldName_1",
+                        "Name_EN",
+                        "Description_EN",
```

Comment:
Ok, will allow additional fields for "ServicePlans" that starts with "Name_" / "Description_"

Reply:
I'll merge the PR, but please check that if any additional column appears we shouldn't fail because of it. The system should be treated as open - which means it retrieves all needed information from the input and ignore others